evap/staff/views.py

```diff
@@ -1418,9 +1421,18 @@ def helper_single_result_edit(request, evaluation):
 
 @require_POST
 @manager_required
+@transaction.atomic
 def evaluation_delete(request):
     evaluation = get_object_from_dict_pk_entry_or_logged_40x(Evaluation, request.POST, "evaluation_id")
 
+    # See comment in helper_evaluation_edit
+    @receiver(RewardPointGranting.granted_by_evaluation_deletion, weak=True)
+    def notify_reward_points(grantings, **_kwargs):
+        logger.info(
```

Comment:
The other `notify_reward_points` methods use `messages.info` instead, is there any reason to use one over the other here?

Reply:
This is an ajax post handler, so the page of the user will not reload (and thus not show messages) when the deletion goes through. If we called messages.info, the message would show on the next page that the user manually goes to, which I'd say is probably surprising/unexpected.

Discussed here before: https://github.com/e-valuation/EvaP/pull/2358#discussion_r1894641174